.github/workflows/ci_linux_x64_clang_asan.yml

```diff
@@ -46,4 +46,11 @@ jobs:
           # Use a modern clang explicitly.
           CC: clang-19
           CXX: clang++-19
-        run: ./build_tools/cmake/build_and_test_asan.sh
+          SCCACHE_AZURE_CONNECTION_STRING: "${{ secrets.AZURE_CCACHE_CONNECTION_STRING }}"
+          SCCACHE_AZURE_BLOB_CONTAINER: ccache-container
+          SCCACHE_CACHE_ZSTD_LEVEL: 10
+          SCCACHE_AZURE_KEY_PREFIX: "ci_linux_x64_clang_asan"
+        run: |
+          source build_tools/cmake/setup_sccache.sh
+          ./build_tools/cmake/build_and_test_asan.sh
+          sccache --show-stats
```

Comment:
Was this run with a warm cache? https://github.com/iree-org/iree/actions/runs/10836779140/job/30071281789?pr=18489#step:5:13931
```
Compile requests                    5370
Compile requests executed           5370
Cache hits                           811
Cache hits (C/C++)                   811
Cache misses                        4543
Cache misses (C/C++)                4543
```
We might need to change the sccache or CMake settings to get a better cache hit rate. What cache hit rates you see when running locally?

The GCP runners using ccache with GCP storage see 99.98% for ASan: https://github.com/iree-org/iree/actions/runs/10835964274/job/30068638038#step:4:9771

Reply:
Oh, much better on the previous run: https://github.com/iree-org/iree/actions/runs/10836229597/job/30069537025#step:5:13890

```
Compile requests                   5349
Compile requests executed          5349
Cache hits                         5335
Cache hits (C/C++)                 5335
Cache misses                          0
Cache timeouts                        0
```

Was the newer run after you manually deleted the cache or something?